joomla/joomla-cms
# [4.0][Aurora] Apply css grid to class rather than body tag

Pull Request for Issue #17426 .

### Summary of Changes
Applies CSS grid to class rather than body tag.

### Testing Instructions
Check frontend is displaying correctly. Set site offline and ensure the offline page displays correctly (card centered in viewport).
